js/ui/tooltips/auto-suggest.js

```diff
@@ -43,7 +52,6 @@ TooltipEngine.classes.autoSuggest = TooltipBase.extend({
             this.updateTooltip(lookupParams);
             this.placeOnScreen();
             event.stopPropagation();
-            ScratchpadAutosuggest.enableLiveCompletion(false);
```

Comment:
LiveCompletion is always off now, so I could remove these.

Reply:
Why did we turn off auto-complete?

Reply:
Good Q. Note that we still pop-up the function arguments hint, but we disabled the auto completion of functions. (Both are handled by ScratchpadAutoSuggest currently)

See comment on https://github.com/Khan/live-editor/blob/a367b6383f9cb1e984935939635e1a80cea5d445/js/ui/tooltip-engine.js#L113 
Here's the commit where I disabled it:
https://github.com/Khan/live-editor/commit/4e028ca5d15d3b062a4a9b38fd4033ddb1a75758

Autosuggest was a hack project, as I recall, so it didn't have time to get well integrated and tested. It'd be better if it could have more time spent on it.